.github/workflows/pr.yml

```diff
@@ -11,16 +11,17 @@ env:
 jobs:
   lint:
     name: Check linting issues
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
         with:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
-      - uses: actions/setup-node@v3
-        with: { node-version: 16, cache: npm }
```

Comment:
From other PR: it’s generally safe these days to not specify the Node version. Major versions don’t break for most users anymore unless you’re using bleeding-edge/experimental features.